### PR TITLE
Pipelines restruct

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: fixtures/exported/10-runtimes.neon [hunk 1]
```diff
@@ -0,0 +1,61 @@
+App\Model\Entity\RuntimeEnvironment:
+	c-gcc-linux:
+		__construct:
+			- c-gcc-linux
+			- "C (GCC)"
+			- C
+			- "[c, h]"
+			- GNU/Linux
+			- C compiled with GCC
+			- "- name: \"source-files\"\r\n  type: \"file[]\"\r\n  value: \"*.{c,h}\""
```
Comment (Teyras) [inline fixtures/exported/10-runtimes.neon]: Couldn't this be done without \r\n?

**Comment by Teyras** [comment]
Great job! I only have two nitpicks. First, could you please squash the WIP commits that concern the migration together? And second, could you export the pipelines *after* the migration as fixtures?

File: fixtures/exported/10-runtimes.neon (hunk 1 above)
Comment (Neloop) [reply fixtures/exported/10-runtimes.neon]: It is generated automatically...

Comment (Teyras) [reply fixtures/exported/10-runtimes.neon]: And can it be generated without \r\n? :P

Comment (Neloop) [reply fixtures/exported/10-runtimes.neon]: Don't know, don't care 😀

**Comment by Neloop** [comment]
Yes I forgot to generate new pipelines, will do tomorrow...

File: fixtures/exported/10-runtimes.neon (hunk 1 above)
Comment (Neloop) [reply fixtures/exported/10-runtimes.neon]: Funny thing is, it was generated at our production server with CentOS

**Pull request merged**